lib/util/http/github.ts

```diff
@@ -223,14 +223,18 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
     return result;
   }
 
-  private async getGraphql<T = unknown>(
+  public async getGraphql<T = unknown>(
```

Comment:
```suggestion
  public async postGraphql<T = unknown>(
```
We do a post internally, so to be consistent.

Reply:
Well, it was named together with `getGraphqlItems()`, so maybe should be changed to something not interferring with HTTP in any way

Reply:
I don't mind, but `get` should be removed as it implies a `GET` request on http layer. So feel free to name it `postGraphql` or simply `graphql`.

@rarkins @JamieMagee Any suggestion here?

Reply:
I'm ok with "graphql". All lower case to match with get, put, etc

Reply:
So, I think it will be single `graphql` to rule them all, single items like `repository` and many like `edges` and `nodes`.